frontend/src/js/user/userSettings.ts

```diff
@@ -4,6 +4,7 @@ const localStorage: Storage = window.localStorage;
 interface UserSettings {
   arePreviousQueriesFoldersOpen: boolean;
   preferredDownloadFormat?: string; // Usually CSV or XLSX
```

Comment:
Maybe we could even rename this ?
```suggestion
  preferredDownloadEnding?: string; // Usually CSV or XLSX
```